components/raftstore-v2/src/raft/peer.rs

```diff
@@ -24,9 +27,19 @@ pub struct Peer<EK: KvEngine, ER: RaftEngine> {
     /// peer list, for example, an isolated peer may need to send/receive
     /// messages with unknown peers after recovery.
     peer_cache: Vec<metapb::Peer>,
+
+    /// Encoder for batching proposals and encoding them in a more efficient way
+    /// than protobuf.
+    raw_write_encoder: Option<RawWriteEncoder>,
```

Comment:
How about `batch_proposal_encoder`? Better avoid using `raw_write` that could be regarded as something related to rawkv.

Reply:
`Raw` here means the codec is not PB, which is the significant reason for better performance. And the encoder can only merge write commands instead of arbitrary proposals. Perhaps `WriteEncoder` can ease confusions.